coderdojochi/views.py

```diff
@@ -409,7 +410,7 @@ def session_sign_up(request, year, month, day, slug, session_id, student_id=Fals
                 else:
                     ip = request.META['REMOTE_ADDR']
 
-                order, created = Order.objects.get_or_create(guardian=guardian, student=student, session=session_obj, ip=ip)
+                order = Order.objects.create(guardian=guardian, student=student, session=session_obj, ip=ip)
```

Comment:
Why did you remove the ability to update the row in case it was already created?